src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -4192,6 +4192,150 @@ def test_aks_nodepool_add_with_artifact_streaming(
             checks=[self.is_empty()],
         )
 
+    @live_only()
```

Comment:
please leave a note to explain why this case need to be marked as `@live_only()`, OR remove this and commit the recording file (find from pipeline artifact once the live test passed)

Reply:
added comments